src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -12,7 +12,7 @@ import recordEvent from '../../../monitoring/record-event';
 import { isLoggedIn } from '../../../user/selectors';
 import { replaceDomainsInData } from '../../../utilities/environment/stagingDomains';
 
-import MegaMenu from '@department-of-veterans-affairs/formation-react/MegaMenu';
+import MegaMenu from '../components/MegaMenu';
```

Comment:
Once this PR is merged, I will open another to remove `MegaMenu` from `formation-react`.